app/config/semi_automated.yml

```diff
@@ -91,6 +91,18 @@ parameters:
         Redirect:
             regex: 'Weiterleitung nach \[\[.*?erstellt'
             label: Weiterleitung
+    ko:
+        Undo:
+            regex: '판 편집을 되돌림'
+            tag: mw-undo
```

Comment:
This gets merged into the global rule, so the tag specification isn't needed here, same with rollback below

Reply:
tag: isn’t correctly catching old (pre-tag era) edit summaries so I thought I would put that there.

Reply:
Correct, but any per-wiki or per-language rules get merged into global ones, e.g. https://github.com/x-tools/xtools/pull/181/files#diff-03baa01bde8a3fd8340f8f6d32967575R34

Adding it here won't hurt anything, though :) The merging thing is just a means to keep the code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Reply:
OK, so I think `tag:` is the only thing that needs to be removed?